README.md

```diff
@@ -68,9 +69,9 @@ documentation: https://experienceleague.adobe.com/docs/experience-manager-cloud-
 
 ## Components
 
-* [Search Redirect](ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/searchredirect/README.md)
-* [Search](ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/search/README.md)
-* [Search Tab](ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/searchtab/README.md)
+1. **[Search:](ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/search/README.md)** Container component which consists of an input field for the search and Search Tabs. Typically used on a Search Result Page   
+2. **[Search Tab:](ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/searchtab/README.md)** Component which performs a search displays the Search Results. 
```

Comment:
I think there's a word missing...
"Component which performs a search _and_ displays the Search Results.", right?